docs/AUTHORING.es.md

```
@@ -0,0 +1,122 @@
+# Creación de plantillas y extensiones
+
+## Estructura del directorio de plantilla
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)